From 906822bb69375c3cb3c2cf0f48cd32df8a4c0f29 Mon Sep 17 00:00:00 2001
From: Jacky Bai <ping.bai@nxp.com>
Date: Wed, 25 Aug 2021 16:39:00 +0800
Subject: [PATCH 35/40] LF-4481-03 arm64: mxc: select the tpm timer by default

Select the tpm timer by default for ARMv8 based i.MX SoC.

Signed-off-by: Jacky Bai <ping.bai@nxp.com>
Reviewed-by: Peng Fan <peng.fan@nxp.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 538cc0c86fcc..c2dd6949d9b8 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -197,6 +197,7 @@ config ARCH_MXC
 	select PM_GENERIC_DOMAINS
 	select SOC_BUS
 	select TIMER_IMX_SYS_CTR
+	select CLKSRC_IMX_TPM
 	help
 	  This enables support for the ARMv8 based SoCs in the
 	  NXP i.MX family.
-- 
2.39.0

